src/utils/colorutils.cpp

```diff
@@ -24,11 +24,8 @@ inline qreal getColorLuma(const QColor& c)
 
 bool ColorUtils::colorIsDark(const QColor& c)
 {
-    bool isWhite = false;
-    if (getColorLuma(c) <= 0.60) {
```

Comment:
Do you know why the previous value was 0.6 and we are now changing to 0.5? Maybe 0.6 looked better with certain colors?

Reply:
Honestly, I'm not sure.
Due to some articles linked by issue [978](https://github.com/flameshot-org/flameshot/issues/978).Seems the threshold should be 0.5 ~ 0.7. Would it be better if I change this back to 0.6 ?

Reply:
I chose 0.5 because of the answers link below:

https://stackoverflow.com/questions/1855884/determine-font-color-based-on-background-color

seems all answers using 0.5

Reply:
Okay great thanks for the additional info.